docs/examples/plotting_functions/heatmap.md

```diff
@@ -128,7 +128,7 @@ ys = range(0, 2π, length=100)
 zs1 = [sin(x*y) for x in xs, y in ys]
 zs2 = [2sin(x*y) for x in xs, y in ys]
 
-my_limits = extrema(hcat(zs1, zs2))  # Get maximum and minimum for all z-values
+my_limits = (-2, 2)  # choose some good limits for sin / 2sin
```

Comment:
maybe not `my_limits` but `joint_limits` or something, and say `# here we pick the limits manually for simplicity instead of computing them` so that it's clear that that would be the normal course of action?

Reply:
Good points. I agree with thoose changes.
The optimal thing would perhaps be a helper function that would find the extremas of any heatmap, contour or contour3D (I think those are the ones this would apply to?) in the given figure.